app/models/mixins/archived_mixin.rb

```diff
@@ -2,11 +2,14 @@ module ArchivedMixin
   extend ActiveSupport::Concern
 
   included do
+    scope :deleted, -> { where(:deleted => true) }
+    scope :not_deleted, -> { where(:deleted => false) }
```

Comment:
`ContainerGroup.all` doesn't need a scope to get both archived & active,
but `ems.container_groups.all` does need `ems.container_groups.unscope(:where => :deleted_on)` which I'd prefer abstracting away as a scope.

But the main reason I'm proposing it is to capture intent. Looking at the source, it'll be impossible to tell the difference between `ContainerGroup.all` that used to mean only active and we never revisited,
vs `ContainerGroup.all` where we deliberately decided we want active & archived.